test/unit/parser-test.js

```diff
@@ -69,7 +69,7 @@ describe('parser', function() {
         expect(dimension, 'dimension with empty string should be 0').to.equal(0);
 
         dimension = parser.parseDimension('35%');
-        expect(dimension, 'dimension with percentage string should be the same').to.equal('35%');
+        expect(dimension, 'dimension with percentage string should be the same').to.equal(35);
```

Comment:
hm this seems like a breaking change based on your parse int in the parent function. the test should not need to be changed to pass unless it's something like the above where we are swapping two falsy valuess

Reply:
done